src/lib/config.js

```diff
@@ -1,20 +1,6 @@
 'use strict'
 
 const Config = require('five-bells-shared').Config
```

Comment:
Hmm, I think it would be better to stick with standard conventions. Otherwise people will get confused thinking it's a class. You could do something like `const configUtils = require(...)` `const config = configUtils.loadConfig(...)`